google/resource_cloudfunctions_function_test.go

```diff
@@ -0,0 +1,476 @@
+package google
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"archive/zip"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"google.golang.org/api/cloudfunctions/v1"
+	"io/ioutil"
+)
+
+const (
+	FUNCTION_TRIGGER_HTTP = iota
+	FUNCTION_TRIGGER_TOPIC
+	FUNCTION_TRIGGER_BUCKET
+)
+
+const testHTTPTriggerPath = "./test-fixtures/cloudfunctions/http_trigger.js"
+const testPubSubTriggerPath = "./test-fixtures/cloudfunctions/pubsub_trigger.js"
+const testBucketTriggerPath = "./test-fixtures/cloudfunctions/bucket_trigger.js"
+
+func TestAccCloudFunctionsFunction_basic(t *testing.T) {
+	t.Parallel()
+
+	var function cloudfunctions.CloudFunction
+
+	funcResourceName := "google_cloudfunctions_function.function"
+	functionName := fmt.Sprintf("tf-test-%s", acctest.RandString(10))
+	bucketName := fmt.Sprintf("tf-test-bucket-%d", acctest.RandInt())
+	zipFilePath, err := createZIParchiveForIndexJs(testHTTPTriggerPath)
+	if err != nil {
+		t.Errorf(err.Error())
+		t.FailNow()
+	}
+	defer os.Remove(zipFilePath) // clean up
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckCloudFunctionsFunctionDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCloudFunctionsFunction_basic(functionName, bucketName, zipFilePath),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCloudFunctionsFunctionExists(
+						funcResourceName, &function),
+					resource.TestCheckResourceAttr(funcResourceName,
+						"name", functionName),
+					//testAccCloudFunctionsFunctionName(functionName, &function),
```

Comment:
this should either be uncommented or removed

Reply:
Removed